Containerfile

```diff
@@ -10,8 +10,9 @@ FROM ${BASE_IMAGE}:${FEDORA_MAJOR_VERSION} AS bluefin
 ARG IMAGE_NAME="${IMAGE_NAME}"
 ARG FEDORA_MAJOR_VERSION="${FEDORA_MAJOR_VERSION}"
 
-COPY etc /etc
 COPY usr /usr
+COPY usr/etc/systemd /etc/systemd
+COPY usr/etc/yum.repos.d /etc/yum.repos.d
```

Comment:
Would it be worth keeping both the `/etc/` and `/usr/` files in the repo for easier management and understanding?

We're chucking files into `/usr/etc` in the repo but that's not where they end up in the image.  It could be confusing when debugging issues

Reply:
I think the idea of having them in `/usr/etc` is that we can reset for a fresh start. So they have to be there for that operation. When you have them in `/etc/` in the repo, changes may be that they don't end up in `/usr/etc/`. 

They are available from that location on boot, but while building the container they are not. That is why I copy them in there explicit.

I will let you all decide what is best, because I am not comfortable enough to make a good descision on this point.

Reply:
Jorge and others are happy with the way you've proposed, so I'm ok with it.  

I just wasn't too sure of how this would be used.

Reply:
Maybe this [ostree deployment](https://ostreedev.github.io/ostree/deployment/) is where the idea also came from. In cominbation with the config reset to get a fresh start.

> Besides the exceptions of /var and /etc then, the rest of the contents of the tree are checked out as hard links into the repository. It’s strongly recommended that operating systems ship all of their content in /usr, but this is not a hard requirement.

Reply:
Since the build is busted anyway, would it help if we document how it should be used? I'm thinking something like "we'll ship the configs in /usr/etc so you have a copy, here's how you override a few of them, like your distrobox.ini, etc?"

Reply:
> Maybe this [ostree deployment](https://ostreedev.github.io/ostree/deployment/) is where the idea also came from. In cominbation with the config reset to get a fresh start.

I think the more relevant part of the link is: https://ostreedev.github.io/ostree/deployment/#contents-of-a-deployment

> The deployment should not have a traditional UNIX /etc; instead, it should include /usr/etc. This is the “default configuration”. When OSTree creates a deployment, it performs a 3-way merge using the old default configuration, the active system’s /etc, and the new default configuration. In the final filesystem tree for a deployment then, /etc is a regular writable directory.


TLDR: The idea of this PR seems good, but after moving files (in repo) from `etc` to `usr/etc`, I think it's contrary to the goal to copy any files back to `/etc/*` during the build. 

Specifically I'm referring to lines like:
```
COPY usr/etc/systemd /etc/systemd
COPY dx/usr/etc/yum.repos.d /etc/yum.repos.d
COPY framework/usr/etc/systemd /etc/systemd
```

Reply:
So maybe after all the best way for now is just leave the repo structure in place and just add a `COPY /etc /usr/etc` instead like @p5 suggested? This way we don't have to think which file goes where when doing both `/etc` and `/usr/etc` at repo level. 

Or maybe even ignore this PR all together because it seems that the files we never put in `/usr/etc/` at this moment are all there to have the original version of the file. Which was our first reason for doing this if i'm correct.